consensus/src/detail/orphan_blocks.rs

```diff
@@ -45,7 +45,7 @@ impl OrphanBlocksPool {
             orphan_by_id: BTreeMap::new(),
             orphan_by_prev_id: BTreeMap::new(),
             max_orphans: DEFAULT_MAX_ORPHAN_BLOCKS,
-            rng: rand::thread_rng(),
+            rng: StdRng::from_entropy(),
```

Comment:
Is there a specific reason we're not using the centralised randomness source in the crypto crate for randomness?

Reply:
Resolving as a duplicate